src/test/kotlin/uk/gov/justice/digital/hmpps/subjectaccessrequestworker/integration/client/DocumentStorageClientIntTest.kt

```diff
@@ -36,10 +44,16 @@ class DocumentStorageClientIntTest : IntegrationTestBase() {
   private lateinit var oAuth2AuthorizedClientService: OAuth2AuthorizedClientService
 
   private val subjectAccessRequestId = UUID.randomUUID()
+  private val subjectAccessRequest = SubjectAccessRequest(id = subjectAccessRequestId)
+
+  @MockBean
+  private lateinit var telemetryClient: TelemetryClient
```

Comment:
we should be using mockito.kotlin so this can be
  private val telemetryClient: TelemetryClient = mock()